splink/aws/aws_linker.py

```diff
@@ -0,0 +1,186 @@
+import sqlglot
```

Comment:
I think it should be called `athena_linker.py` - i.e. let's be as specific as possible with naming

This implies various other renamings e.g. `AWSDataFrame` becomes `AthenaDataFrame` and so on - prob need to find all instances of `aws` to find everything that needs to be renamed